### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by kescobo**
Personnel updates and simplification of issue / PR templates. See https://github.com/orgs/BioJulia/discussions/11#discussioncomment-6629151

- [x] I have updated any relevant documentation and docstrings
- [x] I have added unit tests, and the CodeCov bot shows my new code is covered by tests
- [x] I have mentioned my changes in the CHANGELOG.md file

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -1,40 +1,6 @@
-<!--- Provide a general summary of the issue in the Title above -->
+Thank you for making an issue.
+If you are submitting a bug report, it will help us if you include the following information:
 
-> _This template is rather extensive. Fill out all that you can, if are a new contributor or you're unsure about any section, leave it unchanged and a reviewer will help you_ :smile:. _This template is simply a tool to help everyone remember the BioJulia guidelines, if you feel anything in this template is not relevant, simply delete it._
+- Your version of Julia and {THIS PACKAGE}
```
Comment (jakobnissen) [inline .github/ISSUE_TEMPLATE.md:4]: Maybe better to write "Julia and all used packages", both so we don't forget to replace THIS PACKAGE and so users remember other deps

File: HUMANS.md [hunk 2]
```diff
@@ -2,9 +2,12 @@
 
 ## Contributing Maintainers
 
-- Ben J. Ward
-  - GitHub: [BenJWard](https://github.com/BenJWard)
-  - Email: ben.ward@earlham.ac.uk
-  - Twitter: @Ward9250
+- Kevin Bonham
+  - GitHub: [kescobo](https://github.com/kescobo)
+  - Mastadon: [@Kbonham@scicomm.xyz](https://scicomm.xyz/@Kbonham)
```
Comment (jakobnissen) [inline HUMANS.md:7]: Isn't it Mastodon?

**Review by jakobnissen — APPROVED** [review]
Looks good! Thanks for doing this :)

File: HUMANS.md (hunk 2 above)
Comment (kescobo) [reply HUMANS.md:7]: 😳

**Pull request merged**